README.md

```diff
@@ -164,4 +194,6 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+I believe this would need to be a type 2 Chomsky grammar because the state of 
+the machine is followed as well as what the sentence is, where the next symbol 
+may not be a terminal.
```

Comment:
# 2
Your definition of a type 2 grammar is accurate. But this grammar can be implemented by a state machine with no stack, which means by definition it is regular, or type 3.